hwdb.d/60-keyboard.hwdb

```diff
@@ -1045,8 +1046,9 @@ evdev:input:b0005v046DpB305*
 evdev:input:b0003v046DpB30B*
 evdev:input:b0005v046DpB30B*
  KEYBOARD_KEY_c0183=media	# HUT says consumer control configuration, kbd says Media Center
- KEYBOARD_KEY_c100e=images      # Camera icon, "Photo Gallery"
- KEYBOARD_KEY_c100f=config      # Window with gear icon
+ KEYBOARD_KEY_c100e=images	# Camera icon, "Photo Gallery"
+ KEYBOARD_KEY_c100f=config	# Window with gear icon
+ KEYBOARD_KEY_c102c=fn		# Fn key
```

Comment:
I'd very much prefer to use spaces everywhere. Tabs are bad, even here the file is rendered wrong.